nodebuilder/tests/reconstruct_test.go

```diff
@@ -36,6 +36,7 @@ Steps:
 5. Check that a FN can retrieve shares from 1 to 20 blocks
 */
 func TestFullReconstructFromBridge(t *testing.T) {
+	t.Skip()
```

Comment:
Why these 3 tests?

Reply:
They are temporarily skipped until we come back to reconstruction. Agree though that adding a reason string to Skip would be helpful cc @walldiss